ansible/roles/ocp-workload-starter-workshop/tasks/per_user.yml

```diff
@@ -0,0 +1,77 @@
+---
+- set_fact:
+    my_user: "{{ item }}"
+
+- name: per_user {{my_user}} Tasks Started
+  debug:
+    msg: "per_user {{my_user}} Tasks - Started"
+
+# Create user project and make him admin
+- name: Create workshop user's project for user "{{ my_user }}"
+  shell: oc adm new-project "{{ my_user }}"
+  ignore_errors: yes
+
+- name: Annotate the project as requested by user for user "{{ my_user }}"
+  shell: "oc annotate namespace {{my_user}} openshift.io/requester={{my_user}} --overwrite"
+  ignore_errors: yes
+
+- name: Award admin permission for user "{{ my_user }}"
+  shell: oc adm policy add-role-to-user admin "{{ my_user }}" -n "{{ my_user }}"
+
+# Create cluster quota for the user to prevent abuse
+- name: Create user Quota - clusterresourcequota
+  shell: |
+        oc create clusterquota clusterquota-"{{ my_user }}" \
+        --project-annotation-selector=openshift.io/requester="{{ my_user }}" \
+        --hard requests.cpu="{{quota_requests_cpu}}" \
+        --hard limits.cpu="{{quota_limits_cpu}}"  \
+        --hard requests.memory="{{quota_requests_memory}}" \
+        --hard limits.memory="{{quota_limits_memory}}" \
+        --hard configmaps="{{quota_configmaps}}" \
+        --hard pods="{{quota_pods}}" \
+        --hard persistentvolumeclaims="{{quota_persistentvolumeclaims}}"  \
+        --hard services="{{quota_services}}" \
+        --hard secrets="{{quota_secrets}}" \
+        --hard requests.storage="{{quota_requests_storage}}"
+  ignore_errors: true
```

Comment:
Just checking: did you mean each one of the `ignore_errors` in the playbooks ?

don't we want to know if the command for quotas failed ?